survey/tests/testdump.json

```diff
@@ -379,7 +324,7 @@
   "response": 12,
   "created": "2017-03-22T08:24:43.546Z",
   "updated": "2017-07-10T15:07:35.298Z",
-  "body": "Yës"
+  "body": "Y\u00ebs"
```

Comment:
Probably a problem with the way it's dumped, I need to document this as you said.